packages/mobile/src/invite/saga.ts

```diff
@@ -259,6 +260,24 @@ export function* doRedeemInvite(inviteCode: string) {
   }
 }
 
+export function* skipInvite() {
+  yield take(Actions.SKIP_INVITE)
+  Logger.debug(TAG + '@skipInvite', 'Skip invite action taken')
+  try {
+    Logger.debug(TAG + '@skipInvite', 'Creating new account')
+    yield call(getOrCreateAccount)
+    Logger.debug(TAG + '@skipInvite', 'Marking nux flows as complete')
+    // TODO(Rossy): Remove when import screen is removed
+    yield put(denyImportContacts())
+    yield put(setHasSeenVerificationNux(true))
```

Comment:
Given we have a meaningful `SKIP_INVITE_SUCCESS` action, we could also act on it in the corresponding reducers and atomically set the right state we want for that. 
What do you think?

Reply:
You mean handle that action in the verification reducer? I'd rather not have reducers handle actions from other modules

Reply:
Yes, I know this hasn't been the styled used sofar in the app but having meaningful actions and not just glorified getter/setters actions allows decoupling things as originally intended by Redux.

https://twitter.com/dan_abramov/status/682923564006248448

Reply:
Open to doing this going forward then